assets/app/styles/_core.less

```diff
@@ -8,6 +8,7 @@ html, body {
 
 .console-os {
   background-color: @body-bg;
+  height: ~"calc(100% - 121px)"; // removes scrollbar on pages with less than full viewport content
```

Comment:
Does this spike cpu usage when doing things like scrolling or moving the mouse around over the page? I still have nightmares about performance impacts of `expression()` css rules in IE... want to make sure we're not doing that here. Also, does this work correctly when resizing the browser vertically and/or horizontally?